<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPatientNameDialog</class>
 <widget class="QWidget" name="ModifyPatientNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>249</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="FormLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lastNameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="firstNameLabel">
      <property name="text">
       <string>Pr√©nom</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="firstNameLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lastNameLabel">
      <property name="text">
       <string>Nom de famille</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updatePushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Effectuer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
